apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -474,6 +508,60 @@ void _wt_connect(void *arg)
 	WT_LEAVE;
 }
 
+void _wt_scan_connect(void *arg)
+{
+	WT_ENTER;
+	WT_LOG(TAG, "--> scan_connect\n");
+	g_scan_join = 1;
+	struct wt_options *ap_info = (struct wt_options *)arg;
+	wifi_manager_scan_config_s sconfig;
+	sconfig.channel = 0;
+	sconfig.ssid_length = strlen(ap_info->ssid);
+	strncpy(sconfig.ssid, ap_info->ssid, sconfig.ssid_length + 1);
+	strncpy(g_scanned_ssid, ap_info->ssid, sconfig.ssid_length + 1);
+
+	wifi_manager_result_e res = wifi_manager_scan_ap(&sconfig);
+
+	if (res != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "request scan ssid (%s) fail %d\n", sconfig.ssid, res);
+		return;
+	}
+	WT_TEST_WAIT;
+
+	g_scan_join = 0;
+	if (g_scanned_result != 0) {
+		WT_LOGE(TAG, "scan ssid fail\n");
```

Comment:
Isn't it necessary to call sem_post to unblock sempahore got by calling WT_TEST_WAIT, line 529?

Reply:
_wt_scan_done calls WT_TEST_SIGNAL, so it's ok